src/ToDoItem/ToDoItem.tsx

```diff
@@ -1,66 +1,80 @@
 import React from "react";
 import Checkbox from "@mui/material/Checkbox";
-import { Button, Typography, IconButton } from "@mui/material";
-import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
-import Box from "@mui/material/Box";
+import { Typography, IconButton, Box, Paper } from "@mui/material";
 import DeleteIcon from "@mui/icons-material/Delete";
+import { Draggable } from "react-beautiful-dnd";
+import type { DropResult, DraggableChildrenFn } from "react-beautiful-dnd";
 
 interface ToDoItemProps {
+  id: number;
+  index: number;
   description: string;
   completed: boolean;
   onToggle: () => void;
   onDelete: () => void;
 }
 
 const ToDoItem: React.FC<ToDoItemProps> = ({
+  id,
+  index,
   description,
   completed,
   onToggle,
   onDelete,
 }) => {
-  //todo: use "sx" property instead of classNames + remove 'classnames' node module from package.json
   return (
-    <Box
-      sx={{
-        display: "flex",
-        columnGap: 2,
-        alignItems: "center",
-        fontSize: "18px",
-        width: "100%",
-        justifyContent: "space-between",
-      }}
-    >
-      <Box
-        sx={{
-          display: "flex",
-          alignItems: "center",
-          columnGap: 1,
-        }}
-      >
-        <Checkbox
-          color="info"
-          checked={completed}
-          onChange={onToggle}
+    <Draggable draggableId={id.toString()} index={index}>
+      {(provided: any, snapshot: any) => (
```

Comment:
Do not use `any` type. This is a bad practice.

In the code above types are implicit. That means you don't need to define them explicitly.

But when you will remove those `any`, typescript compiler will start showing errors. That means you did something incorrectly. Please, take a second look on this one.

Reply:
I mean find exactly which causes a problem